addons/mail/static/src/components/chat_window/chat_window.js

```diff
@@ -122,6 +122,9 @@ export class ChatWindow extends Component {
                 this._inputRef.comp.focus();
             }
         }
+        if (!this.root.el) {
+            return;
+        }
```

Comment:
arguable that it could even be moved at the top instead (if there's no root, _inputRef will also be false anyway)